jar-persistence/src/main/java/com/sixsq/slipstream/persistence/Vm.java

```diff
@@ -87,6 +87,18 @@
 	@Attribute(required = false)
 	private String runUuid;
 
+	@Attribute(required = false)
+	private String ip;
+
+	@Attribute(required = false)
+	private String name;
+
+	@Attribute(required = false)
+	private String nodeName;
+
+	@Attribute(required = false)
+	private String nodeInstanceId;
+
```

Comment:
Will DB migration be required? 
Will the attributes be rendered in the document by the serialiser for the old records that don't have values for the fields?